### PR TITLE
Cli & Schematics for Http Entity Service

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
## PR Checklist
- [ X ] The commit message follows our guidelines: https://github.com/datorama/akita/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ X ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Didn't support Http Entity Service

## What is the new behavior?
Cli & Schematics support Http Entity Service

## Does this PR introduce a breaking change?
```
[ ] Yes
[ X ] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "@datorama/akita",
   "version": "4.7.0",
   "description": "State Management Tailored-Made for JS Applications",
-  "schematics": "./schematics/collection.json",
+  "schematics": "./schematics/src/collection.json",
```
Comment (NetanelBasal) [inline package.json]: Why did you change it?

File: schematics/src/ng-add/index.ts [hunk 2]
```diff
@@ -31,6 +42,15 @@ function addPackageJsonDependencies(options: Schema): Rule {
       });
     }
 
+    if (options.entityService) {
+      // @todo: uncomment when lib is on npm
+      // dependencies.push({
+      //   type: NodeDependencyType.Dev,
```
Comment (NetanelBasal) [inline schematics/src/ng-add/index.ts]: NodeDependencyType.Default

File: schematics/src/ng-add/index.ts [hunk 3]
```diff
@@ -148,6 +182,25 @@ function addModuleToImports(options: Schema): Rule {
       addModuleImportToRootModule(host, importRouter, null as any, project);
     }
 
+    if (provideEntityServiceConfig) {
```
Comment (NetanelBasal) [inline schematics/src/ng-add/index.ts]: Why not use `addModuleImportToRootModule`?

File: package.json (hunk 1 above)
Comment (GrandSchtroumpf) [reply package.json]: The path was not correct. The collection.json is inside `schematics/src/`. When linking @datorama/akita I couldn't use the schematics.

Comment (NetanelBasal) [reply package.json]: When we build the project, this is the correct path.

File: schematics/src/ng-add/index.ts (hunk 3 above)
Comment (GrandSchtroumpf) [reply schematics/src/ng-add/index.ts]: It's not a module, it's a provider.

**Comment by NetanelBasal** [comment]
I will merge it when we release the package. Great work.

File: package.json (hunk 1 above)
Comment (GrandSchtroumpf) [reply package.json]: Oh, I see. Sorry about that. I was only building the schematics subfolder when developing I didn't realize. I change that.

File: schematics/src/ng-add/index.ts [hunk 4]
```diff
@@ -31,6 +42,15 @@ function addPackageJsonDependencies(options: Schema): Rule {
       });
     }
 
+    if (options.entityService) {
+      // @todo: uncomment when lib is on npm
+      // dependencies.push({
+      //   type: NodeDependencyType.Default,
+      //   version: 'latest',
```
Comment (NetanelBasal) [inline schematics/src/ng-add/index.ts]: You can remove the comment and change the version to: `^1.0.0`

**Comment by GrandSchtroumpf** [comment]
Hmm, I don't see any button to merge. I think I cannot.
